TypeDBConsole.java

```diff
@@ -152,7 +154,6 @@ private static CLIOptions parseCLIOptions(String[] args) {
                     CLI.usage(CLI.getOut());
                     System.exit(0);
                 } else if (CLI.isVersionHelpRequested()) {
-                    CLI.printVersionHelp(CLI.getOut());
```

Comment:
`CLI.execute()` already prints the version, this statement caused it to be printed a second time.